apps/platform/src/lib/api-functions/secrets.ts

```diff
@@ -0,0 +1,26 @@
+import { z } from 'zod'
+import { zSecret, type Secret } from '@/types'
+import { apiClient } from '../api-client'
+
+async function getAllSecretbyProjectId(
+  projectId: string
+): Promise<Secret[] | undefined> {
+  try {
+    const secretData = await apiClient.get<Secret[]>(`/secret/all/${projectId}`)
```

Comment:
Could you change this to `/secret/${projectId}`? I'm updating the API routes to strictly follow rest conventions.

Reply:
Also, implement the PR suggestions if they seem fit.

Reply:
okey will do that